contrib/grimshot

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+## Grimshot: a helper for screenshots within sway
+## Requirements:
+##  - `grim`: screenshot utility for wayland
+##  - `slurp`: to select an area
+##  - `swaymsg`: to read properties of current window
+##  - `wl-copy`: clipboard utility
+##  - `jq`: json uliity to parse swaymsg output
+##  - `notify-send`: to show notifications
+##  - `mktemp`: to create a temporary file
+## Those are needed to be installed, if unsure, run `grimshot check`
+##
+## Examples:
+## `grimshot copy win` - to copy current window
+## `grimshot save area` - to select area and save it to default file (Pictures/Grimshot-$datetime.png)
+## `grimshot save screen ~/screenshot.png` - to save screenshot under ~/screenshot.png
+## `grimshot` - usage
+## `grimshot check` - verify if tools are installed
+
+ACTION=${1:-usage}
+SUBJECT=${2:-screen}
+FILE=${3:-$(xdg-user-dir PICTURES)/$(date +'Grimshot %Y-%m-%d %H-%M-%S.png')}
+if [ "$ACTION" = "usage" ] ; then
+  echo "Usage:"
+  echo "  grimshot copy|save win|screen|area [FILE]"
+  echo "Troubleshoot:"
+  echo "  grimshot check"
+  exit
+fi
+
+notify() {
+  CMD="notify-send -t 3000 -a grimshot $*"
+  sh -c "$CMD"
```

Comment:
```
notify-send -t 3000 -a grimshot "$@"
```
see https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_05_02

Reply:
:+1: thanks. 
Also
- got rid of `sh -c` invocations 
- assigment `GEOM=-g $GEOM` has been removed

those are now handled by `takeScreenshot`. Looks better, or does it?